docs-partials/builder/vsphere/clone/CustomizeConfig-not-required.mdx

```diff
@@ -4,7 +4,9 @@
 
 - `windows_options` (\*WindowsOptions) - Settings to Windows guest OS customization.
 
-- `windows_sysprep_file` (string) - Supply your own sysprep.xml file to allow full control of the customization process out-of-band of vSphere.
+- `windows_sysprep_file` (string) - Provide a sysprep.xml file to allow control of the customization process independent of vSphere. This option is deprecated, please use windows_sysprep_text.
+
+- `windows_sysprep_text` (string) - Provide the text for the sysprep.xml content to allow control of the customization process independent of vSphere.
```

Comment:
I feel like we need some usage examples here. The ability to use `file()` in HCL is a good example, HCL also supports heredocs but I suspect this would not be easy for a user running JSON templates. JSON templates are legacy but we should call out that this attribute is intended to be used with user variables or HCL expressions that can provide the file contents and not the raw XML. 

What do you think?